indico/modules/events/editing/blueprint.py

```diff
@@ -17,6 +17,9 @@
 _bp.add_url_rule('/editing/api/file-types', 'api_file_types', backend.RHEditingFileTypes)
 _bp.add_url_rule('/editing/api/tags', 'api_tags', backend.RHEditingTags)
 
+
+_bp.add_url_rule('/contributions/<int:contrib_id>/editing/<any(paper):type>/<int:revision_id>/revision-files.zip',
```

Comment:
`.../files.zip`?

Reply:
Hm, no strong opinion. `revision-files.zip` doesn't look bad to me.